### ISSUE TITLE
Website support HTTP/2 and QUIC

### ISSUE THREAD

**Issue by happyman** [happyman/twmap#37]
追一下流行。
chrome://net-internals/#http2
chrome://net-internals/#quic

https://chrome.google.com/webstore/detail/http2-and-spdy-indicator/mpbpobfflnpcgagjijhmgnchggcjblin

ref:
https://tools.keycdn.com/http2-test
performace:
https://tools.pingdom.com/
https://www.webpagetest.org/

**Comment by happyman** [comment]
![image](https://user-images.githubusercontent.com/82296/49334536-5b37df80-f613-11e8-9055-4115ab291ac4.png)
done.

**Comment by happyman** [comment]
note 一下, ubuntu 16.04 更改 mpm, 故 php-fpm 要配合 apache2 設定
/etc/apache2/envvars
換 user
/usr/lib/tmpfiles.d/php7.0-fpm.conf
/etc/php/7.0/fpm/pool.d/www.conf

順便 upgrade aped 到 1.1.2 版
http://ape-project.org/download/ 

解決 aped 出 error_log 
[Mon Dec 03 10:17:23.702885 2018] [proxy_http:error] [pid 28047:tid 139931261179648] (20014)Internal error (specific information not available): [client 163.29.165.106:58244] AH01102: error reading status line from remote server 127.0.0.1:6969, referer: https://map.happyman.idv.tw/twmap/main.php?tab=3
參考:
http://apache-http-server.18135.x6.nabble.com/mod-proxy-error-reading-status-line-td5003910.html
> SetEnv           proxy-initial-not-pooled 1 

This one may - MAY - be useful if your backend aggressively closes 
connections. It prevents a race condition where the backend closes the 
persistent connection after the proxy checks it is alive, but before 
the proxy sends information to the backend. This is quite rare, and if 
your backend did exhibit this behaviour, setting this would have fixed 
it. 

```
<IfModule mod_ssl.c>
<VirtualHost *:443>
   Servername ape.happyman.idv.tw
   ServerAlias *.ape.happyman.idv.tw
   DocumentRoot "/opt/twmap/aped"
         ProxyRequests Off
        <directory proxy:*>
                Order deny,allow
                Deny from all
                Allow from all
        </directory>
        ProxyPass /         http://127.0.0.1:6969/
        ProxyPassReverse /         http://127.0.0.1:6969/
 #  ErrorLog ${APACHE_LOG_DIR}/error.ape.log
 #  CustomLog ${APACHE_LOG_DIR}/access.ape.log combined
  SetEnv proxy-initial-not-pooled 1
    SetEnv proxy-nokeepalive 1

Include /etc/letsencrypt/options-ssl-apache.conf
SSLCertificateFile /etc/letsencrypt/live/map.happyman.idv.tw/fullchain.pem
SSLCertificateKeyFile /etc/letsencrypt/live/map.happyman.idv.tw/privkey.pem
Include /etc/letsencrypt/options-ssl-apache.conf
</VirtualHost>
</IfModule>
```
